.coderabbit.yaml

```
@@ -15,5 +18,9 @@ reviews:
       - develop
       - develop-postgres
       - main
-chat:
-  auto_reply: true
+  path_filters:
+    - '!**/docs/docs/**'
+    - "!*.html"
+    - "!*.md"
+    - "!*.svg" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)